code.json

```diff
@@ -38,12 +38,106 @@
     ],
 
     "contact": {
-      "name": "Jason Laura",
-      "email": "jlaura@usgs.gov"
+      "name": "Adam Paquette",
+      "email": "acpaquette@usgs.gov"
     },
 
     "date": {
-      "metadataLastUpdated": "2023-02-23"
+      "metadataLastUpdated": "2023-04-19"
+    }
+  },
+  {
+    "name": "ale",
+    "organization": "U.S. Geological Survey",
+    "description": "GitLab repository for the Abstraction Layer for Ephemerides package",
+    "version": "0.9.0",
+    "status": "Development",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/ale/-/raw/main/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://code.usgs.gov/astrogeology/ale/",
+    "downloadURL": "https://code.usgs.gov/astrogeology/ale/-/archive/main/ale-main.zip",
+    "disclaimerURL": "https://code.usgs.gov/astrogeology/ale/-/raw/main/DISCLAIMER.md",
```

Comment:
These can't aim at main and have a 0.9.0 version. The version has to aim to where the artifact will be.